docs/sunfish.md

```diff
@@ -0,0 +1,93 @@
+# Installing for the first time and verified boot
+
+It is assumed that you have successfully built your image and signed it with
+your own keys.  Make sure that you know the location of the image and the AVB
+signing key.
+
+ 0. Before you can begin you have to boot the stock OS and go to “Settings /
+    System / Advanced / Developer options” and enable “OEM unlocking”.  On my
+    device I had to insert a SIM card and connect to the network for that, so
+    it looks like you have to connect your device with Google at least once.
```

Comment:
Just a note, the GrapheneOS install instructions say this is part of the "Factory Reset Protection" for devices with Google Play.  If you can find a source for that we could add a mention here.

Reply:
I couldn't find a source so I just linked to the GrapheneOS install instructions.  Additionally I added a link to a German IT privacy blog which suggests that you only need to enable the Captive Portal check (e.g. using a DNS filter like pihole) to make it work.